contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -162,6 +187,7 @@ contract PolicyFacet is IPolicyFacet, XVSRewardsHelper {
      */
     // solhint-disable-next-line no-unused-vars
     function borrowVerify(address vToken, address borrower, uint256 borrowAmount) external {
+        ensureUserAllowed(borrower);
```

Comment:
I think this is not needed, because we are already checking the permission for borrowing in the borrowAllowed check (the "pre-hook")